webapp/javascript/index.jsx

```diff
@@ -48,6 +49,11 @@ ReactDOM.render(
               <AdhocSingle />
             </Route>
           )}
+          {enableAdhoc && (
```

Comment:
IIRC this flag is hardcoded to `true`. You can use `isExperimentalAdhocUIEnabled` instead I think.

Reply:
https://github.com/pyroscope-io/pyroscope/pull/580/commits/6e20912d59bd4369dfecaf3ce6b362d453139eb4